src/data/event.cpp

```diff
@@ -48,6 +49,117 @@ Event Event::fromJson(json j)
     return event;
 };
 
+std::string Event::toBech32Note()
+{
+    char *note = (char *)malloc(KEY_LENGTH*2 + 1);
```

Comment:
I'd prefer we use C++ smart pointers here, if possible, rather than raw `malloc`.